scripts/set_xtensa_params.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright(c) 2023 Intel Corporation. All rights reserved.
+
+set -e
+
+platform=$1
```

Comment:
Add a comment saying this is a non-standard bash extension.

Reply:
Did you mean ${platform,,} ? I removed the lowercase conversion.

Reply:
No, I mean passing an argument to `source script argument` is bash-specific.